.travis.yml

```diff
@@ -109,6 +109,20 @@ jobs:
        - cd ../..
      after_success:
        - make style-py && git diff --exit-code;
+# Julia
+   - name: "Julia"
+     language: julia
+     os: linux
+     julia: 1
+     install:
+       - make info && make -j4
+       - export JULIA_LIBCEED_LIB=$(find $PWD/lib -name "libceed*")
+       - pushd julia/libCEED.jl
+     script:
+       - julia --project -e 'import Pkg; Pkg.build(); Pkg.test("libCEED"; coverage=true)'
```

Comment:
We should also test formatting here.
```

        - julia --project -e 'using Pkg; Pkg.add("JuliaFormatter")'
        - julia --project -e 'using JuliaFormatter; format("src")'
        - git diff --exit-code src/
```

Reply:
Good call. I don't love the choices of JuliaFormatter (e.g. I find `a*x + b` to be much better than `a * x + b`), but it's not a big deal (and some of these things could be customized).

Reply:
Agreed about the default style choices. If you know of good documentation/examples on modifying the style, it would be helpful in my other project. Once we have it dialed in, I like how formatters reduce code churn.

Reply:
It's a bit tricky to customize the styles, it requires manipulating an AST-like object called FST. It is very powerful though. You can check out the script in `julia/libCEED.jl/.CeedStyle.jl`. It's a modification of the default style, but operators with the same precedence as `*` are not automatically surrounded with spaces (e.g. so fractions appear as `1/2` rather than `1 / 2`. Definitely agree about code churn, this should be useful.